Registry/RegistryParse.py

```diff
@@ -322,15 +380,26 @@ def checksum(self):
 
     def validate_checksum(self):
         """
-        Is the file checksum valid.
+        Is the file checksum valid?
         """
         return self.calculate_checksum() == self.checksum()
 
-    def validate(self):
```

Comment:
This removes a public api and might break existing uses,
I believe The def validate() returning bool should be left as is functionally, but marked as obsolete.